src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -67,21 +67,22 @@
         >
           {{ item.index + 1 }} of {{ item.siblingsCount }} symbols inside
         </span>
-        <Reference
+        <OptionalLink
```

Comment:
Did you consider just using the `isGroupMarker` to conditionally use a span or link here directly instead of creating a new "Optional..." component?

I don't know if it would necessarily be better or worse off the top of my head—just wondering if you had already tried that.